lobby.go

```diff
@@ -192,6 +192,50 @@ func wsListen(lobby *Lobby, player *Player, socket *websocket.Conn) {
 									}
 								}
 							}
+						case "setmp":
+							if player == lobby.Owner {
+								newMaxPlayersValue := strings.TrimSpace(command[1])
+								newMaxPlayersValueInt, err := strconv.ParseInt(newMaxPlayersValue, 10, 64)
+								if err == nil {
+									if int(newMaxPlayersValueInt) > len(lobby.Players) && newMaxPlayersValueInt <= lobbySettingBounds.MaxMaxPlayers && newMaxPlayersValueInt >= lobbySettingBounds.MinMaxPlayers {
```

Comment:
Shouldn't it be:

```go
int(newMaxPlayersValueInt) > len(lobby.Players)
```

Since you could also say "Hey I have 5 slots now, but since we are 4 people anyway, I am reducing it to 4."?